packages/react-components/react-drawer/stories/Drawer/index.stories.tsx

```diff
@@ -1,17 +1,20 @@
 import { Drawer } from '@fluentui/react-drawer';
 import descriptionMd from './DrawerDescription.md';
-import DrawerBestpracticesMd from './DrawerBestPractices.md';
+import bestPracticesMd from './DrawerBestPractices.md';
```

Comment:
wondering why the rename ? shouldn't we stick with consistent naming generated by generator ?

Reply:
gotcha, ty!